oshmem/mca/scoll/ucc/scoll_ucc_alltoall.c

```diff
@@ -0,0 +1,75 @@
+/**
+  Copyright (c) 2021 Mellanox Technologies. All rights reserved.
+  $COPYRIGHT$
+
+  Additional copyrights may follow
+
+  $HEADER$
+ */
+#include "scoll_ucc.h"
+#include "scoll_ucc_dtypes.h"
+#include "scoll_ucc_common.h"
+
+#include <ucc/api/ucc.h>
+
+static inline ucc_status_t mca_scoll_ucc_alltoall_init(const void *sbuf, void *rbuf,
+                                                       int count,
+                                                       mca_scoll_ucc_module_t * ucc_module,
+                                                       ucc_coll_req_h * req)
+{
+    ucc_coll_args_t coll = {
+        .mask = 0,
+        .coll_type = UCC_COLL_TYPE_ALLTOALL,
+        .src.info = {
+            .buffer = (void *)sbuf,
+            .count = count,
+            .datatype = UCC_DT_UINT8,
+            .mem_type = UCC_MEMORY_TYPE_UNKNOWN
+        },
+        .dst.info = {
+            .buffer = rbuf,
+            .count = count,
+            .datatype = UCC_DT_UINT8,
+            .mem_type = UCC_MEMORY_TYPE_UNKNOWN
+        },
+    };
+
+    SCOLL_UCC_REQ_INIT(req, coll, ucc_module);
+    return UCC_OK;
+fallback:
+    return UCC_ERR_NOT_SUPPORTED;
+}
+
+
+int mca_scoll_ucc_alltoall(struct oshmem_group_t *group,
+                           void *target,
+                           const void *source,
+                           ptrdiff_t dst, ptrdiff_t sst,
+                           size_t nelems,
+                           size_t element_size,
+                           long *pSync,
+                           int alg)
+{
+    mca_scoll_ucc_module_t *ucc_module;
+    size_t count;
+    ucc_coll_req_h req;
+
+    UCC_VERBOSE(3,"RUNNING UCC ALLTOALL");
```

Comment:
maybe for consistency between components lets write verbose messages in lower case ? (https://github.com/open-mpi/ompi/blob/62362849cae65b2445723e426affc2bb7918a6c8/ompi/mca/coll/ucc/coll_ucc_alltoall.c#L71)